students/stephen/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,20 @@
+# The Classic Fizz Buzz Problem
+# Write a program that prints the numbers from 1 to 100 inclusive.
+# But for multiples of three print “Fizz” instead of the number.
+# For the multiples of five print “Buzz” instead of the number.
+# For numbers which are multiples of both three and five print “FizzBuzz” instead.
+
+def fizzbuzz():
+    for i in range(1,101):
+        if i % 3 == 0 and i % 5 == 0:
```

Comment:
Nice, looks good! If you have extra time, you should check out the one line fizz buzz in the solutions.